src/benchmark/java/com/commercetools/sync/benchmark/ProductSyncBenchmark.java

```diff
@@ -60,6 +60,10 @@ class ProductSyncBenchmark {
   private List<String> errorCallBackMessages;
   private List<String> warningCallBackMessages;
   private List<Throwable> errorCallBackExceptions;
+  private static final int PRODUCT_BENCHMARKS_CREATE_ACTION_THRESHOLD =
+      30_000; // 150% of the maximum value (20K)
```

Comment:
Why is it sometimes 200%, sometimes 150% of the maximum value? Please explain or unify this.

Reply:
Some resources have a threshold with 200% of highest value because the current benchmark already exceeded 150%. That's why I keep the threshold unchanged. Meanwhile some resources keep the benchmark at low level, that's why I adjusted the threshold a bit (as discussed) to 150%.

Anyway, let me unify them to 200% as used to be.

Reply:
> What about this point `Discuss the performance effect of this change and decide with considering tradeoffs.`? What is the build time before and now?
> 
> What about `Update docs/BENCHMARK.md according to changes.`?

Currently benchmarks test elapse about 7-8 minutes. That means the entire build time would be 7-8 minutes longer than it used to be. I think it is acceptable and it helps developers to trace the performance change for every commit.

The documentation has been modified here
https://github.com/commercetools/commercetools-sync-java/pull/868/commits/398f735c6db05c425165a4453721f37309776fe7